spack_dlaf/packages/dla-future/package.py

```diff
@@ -6,7 +6,7 @@ class DlaFuture(CMakePackage):
     homepage = "https://github.com/eth-cscs/DLA-Future.git/wiki"
     git      = "https://github.com/eth-cscs/DLA-Future.git"
 
-    maintainers = ['Sely85']
+    maintainers = ['Sely85', 'teonnik']
```

Comment:
I've only made the first draft of all these packages, please at least put your name first. 
```suggestion
    maintainers = ['teonnik', 'Sely85']
```